src/include/btree_cmp_inline.h

```diff
@@ -119,6 +120,26 @@ __wt_compare(WT_SESSION_IMPL *session, WT_COLLATOR *collator, const WT_ITEM *use
     return (collator->compare(collator, &session->iface, user_item, tree_item, cmpp));
 }
 
+/*
+ * __check_fixed_string --
+ *     Check if the schema format is a fixed string.
+ */
+static inline int
+__check_fixed_string(const char *format)
+{
+    size_t len;
+    const char *p;
+
+    p = format;
+    /* Only check characters before the NULL character at the end */
+    len = strlen(format) - 1;
```

Comment:
I don't really agree with the current implementation. If `strlen(format)` returns 0, `len` will have a big value. And the current implementation does not protect you from something like `10s10s`. I would suggest the following:
```
    size_t len;
    const char *p;

    len = strlen(str);
    p = str;

    for (; len > 1; --len, ++p)
        if (!isdigit(*p))
            return (false);

    return (len == 1 && *p == 's');
```

Reply:
thanks